docs/troubleshooting.md

```diff
@@ -18,13 +18,249 @@
 -->
 
 # Troubleshooting
-This tutorial is designed to provide troubleshooting for end users and developers 
+This tutorial is designed to provide troubleshooting for end users and developers
 who are building, deploying, and using CarbonData.
 
-### General Prevention and Best Practices
- * When trying to create a table with a single numeric column, table creation fails: 
-   One column that can be considered as dimension is mandatory for table creation.
-         
- * "Files locked for updation" when same table is accessed from two or more instances: 
-    Remove metastore_db from the examples folder.
+## Failed to load thrift libraries
 
+  **Symptom**
+
+  Thrift throws following exception :
```

Comment:
For thrift configuration issues or installation problems, it is better to remind users to follow Apache thrift docs : https://thrift.apache.org/docs/install